### PR TITLE
Add interface cases with acpi

### PR DESCRIPTION
**Pull request by chloerh**
- RHEL-279587 - [acpi] Set ACPI index for interfaces
- RHEL-281192 - [acpi] Set ACPI index for interfaces:boundary/negative
values

Signed-off-by: Haijiao Zhao <haizhao@redhat.com>

### CODE REVIEW

**Comment by chloerh** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/3230

File: libvirt/tests/src/virtual_network/network_misc.py [hunk 1]
```diff
@@ -121,8 +127,91 @@ def run_test_zone_info():
         test_zone = eval('run_test_zone_%s' % case)
         test_zone()
 
+    def run_test_iface_acpi(case):
+        """
+        Test acpi setting of interface
+
+        :param case: test case
+        """
+        if not utils_misc.compare_qemu_version(6, 1, 0) and \
```
Comment (Yingshun) [inline libvirt/tests/src/virtual_network/network_misc.py:136]: Do you need to add 'is_rhev=False' to utils_misc.compare_qemu_version()?
qemu-kvm-6.1.0-2.el9.x86_64 was installed on my local env, and the results are as follow:
```
>>> utils_misc.compare_qemu_version(6, 1, 0)
False
>>> utils_misc.compare_qemu_version(6, 1, 0, is_rhev=False)
True
>>> 
```

**Review by yalzhang — APPROVED** [review]
Have ran the cases and checked the logs, pass.

File: libvirt/tests/src/virtual_network/network_misc.py (hunk 1 above)
Comment (chloerh) [reply libvirt/tests/src/virtual_network/network_misc.py]: Thank you! Updated.

**Review by Yingshun — APPROVED** [review]

**Pull request merged**